### PR TITLE
[TEC-5062] Fix deprecation notice

### PR DESCRIPTION
**Pull request by codingmusician**
### 🎫 Ticket
[TEC-5062]

### 🗒️ Description
User wrote in regarding a PHP 8.2 deprecation notice they kept seeing. We were passing an object into the `sanitize_title()` function, which is expecting a string. I was unable to replicate the notice for some reason, but digging into `WP_Theme()` class, I did find that there is a `__toString()` method ([link](https://developer.wordpress.org/reference/classes/wp_theme/__tostring/)) that handles things when casting the object into a string.

### 🎥 Artifacts 
N/A

### ✔️ Checklist
- [x] Ran `npm run changelog` to add changelog file(s). More info [here](https://docs.theeventscalendar.com/developer/git/changelogs/#process)
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [x] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [x] Are all the **required** tests passing?
- [x] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [x] Check the base branch for your PR.
- [x] Add your PR to the project board for the release.

[TEC-5062]: https://stellarwp.atlassian.net/browse/TEC-5062?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tribe/Views/V2/Template_Bootstrap.php [hunk 1]
```diff
@@ -423,7 +424,7 @@ public function filter_add_body_classes( $classes ) {
 			return $classes;
 		}
 
-		$classes[] = 'page-template-' . sanitize_title( $active_theme );
+		$classes[] = 'page-template-' . sanitize_title( (string) $active_theme );
```
Comment (Camwyn) [inline src/Tribe/Views/V2/Template_Bootstrap.php:427]: I think we should be more explicit here and "show our work" but I can be overruled.
```suggestion
		$classes[] = 'page-template-' . sanitize_title( (string) $active_theme->display( 'Name' );
```

Comment (Camwyn) [reply src/Tribe/Views/V2/Template_Bootstrap.php:427]: [wp_get_theme](https://github.com/WordPress/wordpress-develop/blob/6.6.2/src/wp-includes/theme.php#L117-L134) is _always_ going to return an object

Comment (codingmusician) [reply src/Tribe/Views/V2/Template_Bootstrap.php:427]: When a class has a `__toString()` method, it handles being cast to a string. WP_Theme has that method [here](https://developer.wordpress.org/reference/classes/wp_theme/__tostring/).

File: src/Tribe/Views/V2/Template_Bootstrap.php [hunk 2]
```diff
@@ -423,7 +424,7 @@ public function filter_add_body_classes( $classes ) {
 			return $classes;
 		}
 
-		$classes[] = 'page-template-' . sanitize_title( $active_theme );
+		$classes[] = 'page-template-' . sanitize_title( (string) $active_theme->display( 'Name' );
```
[the remark this answers is not in the document]
Comment (Camwyn) [reply src/Tribe/Views/V2/Template_Bootstrap.php:427]:
```suggestion
		$classes[] = 'page-template-' . sanitize_title( (string) $active_theme->display( 'Name' ) );
```

[the remark this answers is not in the document]
Comment (Camwyn) [reply src/Tribe/Views/V2/Template_Bootstrap.php:427]: That as a typo in my suggested code 😆

[the remark this answers is not in the document]
Comment (codingmusician) [reply src/Tribe/Views/V2/Template_Bootstrap.php]: There is also a typo in you correction. LOL

**Review by Camwyn — APPROVED** [review]

**Pull request merged**